tests/auth-react/fastapi-server/app.py

```diff
@@ -44,7 +46,7 @@
     thirdpartypasswordless,
     userroles,
 )
-from supertokens_python.recipe.dashboard import DashboardRecipe
+from supertokens_python.recipe.dashboard.recipe import DashboardRecipe
```

Comment:
Reset unrelated changes in this file

Reply:
`dashboard/__iniy__.py` doesn't globally import `dashboard/recipe.py` so these imports change